packages/p2p/src/components/modal-manager/modals/quick-add-modal/quick-add-modal-buttons/quick-add-modal-buttons.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Button } from '@deriv/components';
+import { observer } from '@deriv/stores';
+import { localize } from 'Components/i18next';
+
+type TQuickAddModalButtonsProps = {
+    className?: string;
+    is_disabled: boolean;
+    onCancel: () => void;
+    onClickAdd: () => void;
+};
+
+const QuickAddModalButtons = ({ className, is_disabled, onCancel, onClickAdd }: TQuickAddModalButtonsProps) => {
+    return (
+        <React.Fragment>
+            <Button has_effect large onClick={onCancel} secondary text={localize('Cancel')} />
+            <Button
+                className={className}
+                has_effect
+                is_disabled={is_disabled}
+                large
+                onClick={onClickAdd}
+                primary
+                text={localize('Add')}
```

Comment:
can we use the <Localize /> component here instead?

Reply:
just checked this, I think in this case its better to use `localize` because it explicitly returns as a string versus `Localize` which returns an Element. Localize would be better if its wrapped using 'Text` component in this case. also i got an error if we use Localize here in the button component because its trying to change the string a bit in the component, and it won't work if Localize is passed